cmake/Modules/ObsDefaults_Linux.cmake

```diff
@@ -0,0 +1,179 @@
+        "${OBS_LIBRARY_DESTINATION}/obs-scripting")
+    set(OBS_DATA_DESTINATION "${CMAKE_INSTALL_DATAROOTDIR}/obs")
+    set(OBS_CMAKE_DESTINATION "${OBS_LIBRARY_DESTINATION}/cmake")
+
+    set(OBS_INSTALL_PREFIX "${CMAKE_INSTALL_PREFIX}/")
+    set(OBS_DATA_PATH "${OBS_DATA_DESTINATION}")
+
+    set(OBS_SCRIPT_PLUGIN_PATH
+        "${CMAKE_INSTALL_PREFIX}/${OBS_SCRIPT_PLUGIN_DESTINATION}")
+    set(CMAKE_INSTALL_RPATH
+        "${CMAKE_INSTALL_PREFIX}/${OBS_LIBRARY_DESTINATION}")
+  else()
+    set(OBS_EXECUTABLE_DESTINATION "bin/${_ARCH_SUFFIX}bit")
+    set(OBS_INCLUDE_DESTINATION "include")
+    set(OBS_LIBRARY_DESTINATION "bin/${_ARCH_SUFFIX}bit")
+    set(OBS_PLUGIN_DESTINATION "obs-plugins/${_ARCH_SUFFIX}bit")
+    set(OBS_SCRIPT_PLUGIN_DESTINATION "data/obs-scripting/${_ARCH_SUFFIX}bit")
+    set(OBS_DATA_DESTINATION "data")
+    set(OBS_CMAKE_DESTINATION "cmake")
+
+    set(OBS_INSTALL_PREFIX "")
+    set(OBS_DATA_PATH "../../${OBS_DATA_DESTINATION}")
+
+    set(OBS_SCRIPT_PLUGIN_PATH "../../${OBS_SCRIPT_PLUGIN_DESTINATION}")
+    set(CMAKE_INSTALL_RPATH
+        "$ORIGIN/" "${CMAKE_INSTALL_PREFIX}/${OBS_LIBRARY_DESTINATION}")
+  endif()
+
+  if(BUILD_FOR_PPA)
+    set_option(ENABLE_LIBFDK ON)
+    set_option(ENABLE_JACK ON)
+    set_option(ENABLE_RTMPS ON)
+  endif()
+
+  if(BUILD_FOR_DISTRIBUTION OR DEFINED ENV{CI})
+    set_option(ENABLE_RTMPS ON)
+  endif()
+
+  set(CPACK_PACKAGE_NAME "obs-studio")
+  set(CPACK_PACKAGE_VENDOR "${OBS_WEBSITE}")
+  set(CPACK_DEBIAN_PACKAGE_MAINTAINER "${OBS_COMPANY_NAME}")
+  set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "${OBS_COMMENTS}")
+  set(CPACK_RESOURCE_FILE_LICENSE
+      "${CMAKE_SOURCE_DIR}/UI/data/license/gplv2.txt")
+  set(CPACK_PACKAGE_VERSION "${OBS_VERSION_CANONICAL}-${OBS_BUILD_NUMBER}")
+  set(CPACK_STRIP_FILES "bin/obs" "bin/obs-ffmpeg-mux")
+  set(CPACK_SOURCE_STRIP_FILES "")
+  set(CPACK_PACKAGE_EXECUTABLES "obs")
+
+  if(OS_LINUX AND NOT LINUX_PORTABLE)
+    set(CPACK_GENERATOR "DEB")
+    set(CPACK_DEBIAN_PACKAGE_SHLIBDEPS ON)
+    set(CPACK_SET_DESTDIR ON)
+  elseif(OS_FREEBSD)
+    option(ENABLE_CPACK_GENERATOR
+           "Enable FreeBSD cpack generator (experimental)" OFF)
+
+    if(ENABLE_CPACK_GENERATOR)
+      set(CPACK_GENERATOR "FreeBSD")
+    endif()
+
+    set(CPACK_FREEBSD_PACKAGE_DEPS
+        "multimedia/v4l_compat"
+        "devel/swig"
+        "devel/cmake"
+        "devel/ninja"
+        "devel/pkgconf"
+        "lang/python37"
+        "x11/xorgproto"
+        "x11/libICE"
+        "x11/libSM"
+        "x11/libX11"
+        "x11/libxcb"
+        "x11/libXcomposite"
+        "x11/libXext"
+        "x11/libXfixes"
+        "x11/libXinerama"
+        "x11/libXrandr"
+        "devel/qt5-buildtools"
+        "devel/qt5-qmake"
+        "lang/python37"
+        "graphics/qt5-imageformats"
+        "multimedia/ffmpeg"
+        "ftp/curl"
+        "devel/dbus"
+        "audio/fdk-aac"
+        "x11-fonts/fontconfig"
+        "print/freetype2 devel/jansson"
```

Comment:
Why are two packages included on the same line?

Reply:
Oversight, fixed.